packages/backend/src/apps/vault-workspace/__tests__/actions/get-data-out-metadata.test.ts

```diff
@@ -0,0 +1,65 @@
+import { IExecutionStep } from '@plumber/types'
+
+import { describe, expect, it } from 'vitest'
+
+import getDataOutMetadata from '../../actions/get-table-data/get-data-out-metadata'
+import { VAULT_ID } from '../../common/constants'
+
+function convertKeyToHex(key: string) {
+  return Buffer.from(key).toString('hex')
+}
+
+describe('Test getDataOutMetadata', () => {
+  it('vault metadata example that has not dataOut (should not occur)', async () => {
```

Comment:
grammar

Reply:
```suggestion
  it('vault metadata example that has not dataOut should not occur', async () => {
```
nit: strings in `it` calls should be readable sentences

Reply:
thanks, will make the change!